buildenv/1.3.x/win32-static/setup/cygwin.cmd

```diff
@@ -9,23 +9,30 @@
 
 @echo off
 
+set MUMBLE_CYGWIN_ROOT_SOURCE=environment
 :: First, try to query the registry for a potential Cygwin
 :: installation directory.
-for /f "tokens=2* delims= " %%a in ('reg query "HKCU\Software\Cygwin\Installations" 2^>nul') do set MUMBLE_CYGWIN_ROOT=%%b
+if not defined MUMBLE_CYGWIN_ROOT (
+	for /f "tokens=2* delims= " %%a in ('reg query "HKCU\Software\Cygwin\Installations" 2^>nul') do set MUMBLE_CYGWIN_ROOT=%%b
+	set MUMBLE_CYGWIN_ROOT_SOURCE=registry
+)
 
 :: HKCU\Software\Cygwin\Installations has a weird prefix on the install dir,
 :: so strip it if it's there.
 if "%MUMBLE_CYGWIN_ROOT:~0,4%" == "\??\" (
 	set MUMBLE_CYGWIN_ROOT=%MUMBLE_CYGWIN_ROOT:~4%
 )
 
-:: The registry query worked. Check if the directory actually
-:: exists. If it doesn't, unset MUMBLE_CYGWIN_ROOT such that
+:: The registry query worked. Might just be cygwin1.dll. Check
```

Comment:
I think the cygwin1.dll thing needs more explanation... How about something like this?

    :: The registry query worked. However, the directory we found might not be a real
    :: Cygwin installation. It might just be an SDK (such as the Android SDK), or something
    :: else that uses a cygwin-1.dll. To determine if it is a proper Cygwin installation, we
    :: check if /bin/bash.exe exists. If it doesn't, unset MUMBLE_CYGWIN_ROOT such
    :: that the next check (the check below us) will fall back to using standard Cygwin
    :: directories.